SpatialGDK/Source/SpatialGDK/Private/Utils/InterestFactory.cpp

```diff
@@ -113,27 +113,17 @@ Interest InterestFactory::CreateServerWorkerInterest()
 
 Interest InterestFactory::CreateInterest() const
 {
-	if (GetDefault<USpatialGDKSettings>()->bEnableServerQBI)
+	if (Actor->IsA(APlayerController::StaticClass()))
 	{
-		if (Actor->GetNetConnection() != nullptr)
```

Comment:
So it was determined you only need interest on the player controller? Not on anything the player owns?

Reply:
From talking with @m-samiec and @MatthewSandfordImprobable this is what I think we settled on, so tagging them just in case I misunderstood that conversation

Reply:
Yes, for client QBI this only needs to be set once.